usr/share/rear/lib/layout-functions.sh

```diff
@@ -530,8 +530,7 @@ get_disk_size() {
 
     local block_size=$(get_block_size ${disk_name%/*})
 
-    [ -r /sys/block/$disk_name/size ]
-    BugIfError "Could not determine size of disk $disk_name, please file a bug."
+    $(retry_command test -r /sys/block/$disk_name/size) || Error "Could not determine size of disk $disk_name"
```

Comment:
Maybe I am looking at the wrong file, what I see is this in line 533 of layout-functions.sh in your PR:
```bash
 $(retry_command test -r /sys/block/$disk_name/size) || Error "Could not determine size of disk $disk_name"
```

So there is no variable that receives the output of the `retry_command` function.